### PR TITLE
Two minor patches

### PR DESCRIPTION
**Pull request by jarrpa**
Incidental stuff I found while working on #355. Not required for it, but I figured I may as well see if they're worth taking anyway. :)

### CODE REVIEW

File: pkg/utils/bodystring.go [hunk 1]
```diff
@@ -18,14 +18,14 @@ package utils
...
-	"io"
 	"io/ioutil"
 	"net/http"
+	"strings"
 )
 
 // Return the body from a response as a string
 func GetStringFromResponse(r *http.Response) (string, error) {
-	body, err := ioutil.ReadAll(io.LimitReader(r.Body, r.ContentLength))
+	body, err := ioutil.ReadAll(r.Body)
```
Comment (lpabon) [inline pkg/utils/bodystring.go]: We need the limit to make sure we do not read forever.  So this line does not need to be fixed.

File: pkg/utils/bodystring.go [hunk 2]
```diff
@@ -39,5 +39,5 @@ func GetErrorFromResponse(r *http.Response) error {
 	if err != nil {
 		return err
 	}
-	return errors.New(s)
+	return errors.New(strings.TrimSpace(s))
```
Comment (lpabon) [inline pkg/utils/bodystring.go]: This is nice!

File: apps/glusterfs/app_node_test.go [hunk 3]
```diff
@@ -352,7 +352,6 @@ func TestNodeAddDelete(t *testing.T) {
 	r, err = http.Post(ts.URL+"/nodes", "application/json", bytes.NewBuffer(request))
 	tests.Assert(t, err == nil)
 	tests.Assert(t, r.StatusCode == http.StatusConflict)
-	tests.Assert(t, err == nil)
```
Comment (lpabon) [inline apps/glusterfs/app_node_test.go]: Nice catch

**Comment by jarrpa** [comment]
@lpabon correctly identified the need to still specify r.ContentLength, and I verified the lack of a guarantee from ReadAll that it wouldn't hoard memory. Further, io.LimitReader will return 0, EOF if it's limit <= 0. Removed that commit. :)

**Comment by lpabon** [comment]
Looks good!

**Pull request merged**